### DISCUSSION TITLE
Application reject can't reapply and facing problem creating a ticket for it.

### DISCUSSION THREAD

**Discussion by VenusCodes** [community/community#28293] [GitHub Education]
My application for student developer pack was rejected, and i wan't to reapply , the issue i am facing is that github want's the supplementary document to be an dated student id which in our school are renewed annually and since the pandemic it's not issued to us . I have an student portal screenshot and an bonafide certificate from the school. Upon trying to create a support ticket for it. It is always generated as an close ticket. Can anyone help me with it ?? 

These are the two support tickets.

https://support.github.com/ticket/1728898

and 

https://support.github.com/ticket/1730392

**Comment by mizzleinetimi** [comment]
i have THE same issue!
